gui/src/components/settings/pages/GeneralSettings.tsx

```diff
@@ -435,6 +436,8 @@ export function GeneralSettings() {
   //   }
   // }, [state]);
 
+  const [skipDriftCompWarning, setSkipDriftCompWarning] = useState(false);
```

Comment:
The name of this doesn't really fit the behaviour, this could follow the implementation for the hand warning
> If null, we still haven't shown the hands warning if false then initially the hands warning was disabled

Or just rename to `showDriftCompWarning`, I think it'd be worthwhile to just show it all the time anyways